src/test/java/jenkins/plugins/office365connector/helpers/AffectedFileBuilder.java

```diff
@@ -0,0 +1,63 @@
+package jenkins.plugins.office365connector.helpers;
+
+import static org.powermock.api.mockito.PowerMockito.mock;
+import static org.powermock.api.mockito.PowerMockito.when;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+
+import hudson.model.AbstractBuild;
+import hudson.model.User;
+import hudson.scm.ChangeLogSet;
+import hudson.scm.EditType;
+import org.mockito.invocation.InvocationOnMock;
+import org.mockito.stubbing.Answer;
+
+/**
+ * @author Damian Szczepanik (damianszczepanik@github)
+ */
+public class AffectedFileBuilder {
+
+    private static final String DEVELOPER_NAME = "Mike";
+
+    private class File implements ChangeLogSet.AffectedFile {
+
+        @Override
+        public String getPath() {
+            return null;
```

Comment:
must be implemented but I don't validate it as I need only number of files, not its content